### ISSUE TITLE
Memory error with high number of observations

### ISSUE THREAD

**Issue by GWeindel** [strengejacke/ggeffects#41]
Hi,
I'm trying to plot the fixed effects of a lmer model, but the number of data points seems to high for ggeffects despite being a reasonnalby large dataset.

reproducible example :
```R
library(lme4)
data = data.frame(X=rnorm(30000), Y=rnorm(30000), Z=rep(1:2),cell=rep(c(1:10),3000))
LME <- lmer(X ~ Y * Z + (1|cell), data=data)
library(ggeffects)
dat <- ggpredict(LME, terms = c("Y","Z"))
plot(dat)
```
`Erreur : impossible d'allouer un vecteur de taille 26.8 Go`

sessionInfo() : 
```R 
R version 3.3.2 (2016-10-31)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 16.04.4 LTS
ggeffects_0.3.1 lme4_1.1-15 
```

Is there a way to reduce the object size ?

Thanks for your package by the way !

**Comment by strengejacke** [comment]
Which package version do you have? This issue should be solved with version >= 0.5.0

**Comment by GWeindel** [comment]
Oh sorry you're right. Works like a charm after uploading to the last github version. Thanks for already having fixed it.
